core/src/main/java/com/graphhopper/routing/DijkstraBidirectionRef.java

```diff
@@ -42,8 +42,8 @@
     protected SPTEntry currFrom;
     protected SPTEntry currTo;
     protected PathBidirRef bestPath;
-    private PriorityQueue<SPTEntry> pqOpenSetFrom;
-    private PriorityQueue<SPTEntry> pqOpenSetTo;
+    protected PriorityQueue<SPTEntry> pqOpenSetFrom;
+    protected PriorityQueue<SPTEntry> pqOpenSetTo;
```

Comment:
Can we make it package protected instead? As it is not final this is a bit unsafe to allow subclasses access to it.